CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix Teams' theme list item end media styles @mnajdova ([#1448](https://github.com/stardust-ui/react/pull/1448))
 - Fix the order of the fela plugin @mnajdova ([#1461](https://github.com/stardust-ui/react/pull/1461))
 
+### Performance
+- Use minified version of `normalize.css` @layershifter  ([#1476](https://github.com/stardust-ui/react/pull/1476))
```

Comment:
You also upgraded normalize from 8.0.0 to 8.0.1. Should it be mentioned in the changelog?

Reply:
Updated the entry 👍